index.js

```diff
@@ -32,12 +32,22 @@ const reporter = () => {
 
     output.write('\n' + chalk.white(assert.name) + '\n')
     if (assert.diag) {
-      const d = assert.diag
-      output.write(chalk.gray(
-        `\toperator: ${d.operator}\n` +
-        `\texpected: ${util.inspect(d.expected)}\n` +
-        `\tactual:   ${util.inspect(d.actual)}\n`
-      ))
+        const d = assert.diag
```

Comment:
The indentation does not match the rest of the file. (Which in turn doesn't match the `.editorconfig`. 😊)